Apps/W1/ShopifyConnector/app/Shipping/Codeunits/ShpfyExportShipments.Codeunit.al

```diff
@@ -0,0 +1,148 @@
+/// <summary>
+/// Codeunit Shpfy Export Shipments (ID 30190).
+/// </summary>
+codeunit 30190 "Shpfy Export Shipments"
+{
+    Access = Internal;
+
```

Comment:
This is correct, because the permissions are already in the role of "D365 BASIC"
Previous we use the permission sets in an xml file, because the permission objects didn't exists.
With the XML file, we must also included these permissions, but now with the permission set objects is this not necessary anymore.

Reply:
Updated with commit: "Inderect Permissions"